datadog_checks_dev/datadog_checks/dev/tooling/release.py

```diff
@@ -127,6 +127,9 @@ def update_agent_requirements(req_file, check, newline):
         if current_package_name == package_name:
             lines[i] = f'{newline}\n'
             break
+    else:
```

Comment:
Which python versions must we support? Can we take advantage of built-in dictionary ordering to make this whole function a bit simpler?

```python
pkg_lines = {line.split('==')[0]: line for line in lines}
pkg_lines[package_name] = f'{newline}\n'
```

Reply:
Actually, if we're sorting the lines at the end of the day, we don't care about python version :)

Reply:
Python 3.8

This a good idea. Thank you.